src/display/editor/annotation_editor_layer.js

```diff
@@ -440,6 +440,16 @@ class AnnotationEditorLayer {
     editor.onceAdded();
   }
 
+  /**
+   * Add an editor in the annotation storage.
+   * @param {AnnotationEditor} editor
+   */
+  addToAnnotationStorage(editor) {
+    if (!editor.isEmpty() && !this.annotationStorage.has(editor.id)) {
```

Comment:
I wonder if we should perhaps use the existing `getRawValue` method here, rather than add a new method (i.e. `has`) with a single call-site?

Reply:
I benchmarked `!!map.get` vs `map.has` and with Firefox `has` is 10% faster when with Chrome it's 25% faster.
Anyway, it isn't really a bottleneck, so I think it doesn't really matters, it's likely a matter of taste here.
If you prefer using `getRawValue`, it's fine for me, just tell me.